### PR TITLE
openjdk: symlink manpages

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

See Homebrew/discussions#1843.

While we're here, let's clean up unnecessary string interpolation
because it's less readable.

### CODE REVIEW

**Comment by carlocab** [comment]
I removed the `CI-build-dependents-from-source` flag. I don't think this is needed after adding a symlink to the bottle.

File: Formula/openjdk.rb [hunk 1]
```diff
@@ -147,15 +148,16 @@ def install
...
 
     on_linux do
       libexec.install Dir["build/linux-x86_64-server-release/images/jdk/*"]
-      bin.install_symlink Dir["#{libexec}/bin/*"]
-      include.install_symlink Dir["#{libexec}/include/*.h"]
-      include.install_symlink Dir["#{libexec}/include/linux/*.h"]
+      bin.install_symlink Dir[libexec/"bin/*"]
+      include.install_symlink Dir[libexec/"include/*.h"]
+      include.install_symlink Dir[libexec/"include/linux/*.h"]
+      man.install_symlink Dir[libexec/"man/*"]
```
Comment (cho-m) [inline Formula/openjdk.rb:160]: CI failure probably due to nested directory structure
```
Could not symlink share/man/man1
```

One option would be to identify all `man*`, e.g.
```suggestion
      man1.install_symlink Dir[libexec/"man/man1/*"]
```

Or perhaps do a loop on all subdirs `Dir[libexec/"man/*"].each do |man|`...

**Comment by carlocab** [comment]
There are a bunch of errors from dependents, but they're surely not related to this change.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Does this also need to be done for `@11` and `@8`?

**Comment by carlocab** [comment]
Being done for `@11` in #80928. I'll look into `@8`.

**Pull request closed without merging**